opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1325,6 +1325,11 @@ Defaulted grid coordinates is not allowed for COMPDAT as part of ACTIONX)"
         return this->snapshots[timeStep].groups.get(groupName);
     }
 
+    bool Schedule::hasChoke(const std::string& nodeName, std::size_t timeStep) const {
```

Comment:
When reviewing the downstream PR, I saw there is the following function, 
```c++
bool Group::as_choke() const {
    return this->m_choke_group.has_value();
}
```

Does this function overlap with the new function introduced here?

Reply:
But it looks like updated in the function `handleNODEPROP()`, which is for extended network. 

Potentially we can reuse `as_choke()` function instead of introducing a new one?